### PR TITLE
Update readme files

### PR DESCRIPTION
**Pull request by daniel-cit**
Update/Fix issue in README files:

- Markdown lint fixes
- Added missing links to inner READMEs that have the variables explanation
- Fixed typos
- Alternative instructions to obtain the service perimeter names in 4-projects.

### CODE REVIEW

**Review by bharathkkb — APPROVED** [review]
LGTM
/cc @rjerrems for another pair of eyes

File: 1-org/README.md [hunk 1]
```diff
@@ -15,32 +15,32 @@ Enabling Data Access logs might result in your project being charged for the add
...
+1. Check if your organization already has a Access Context Manager Policy `gcloud access-context-manager policies list --organization YOUR_ORGANIZATION_ID --format="value(name)"`.
+1. Rename `./envs/shared/terraform.example.tfvars` to `./envs/shared/terraform.tfvars` and update the file with values from your environment and bootstrap step (you can re-run `terraform output` in the 0-bootstrap directory to find these values, the `terraform_service_account` variable is the bootstrap `terraform_sa_email` output). Make sure that `default_region` is set to a valid [BigQuery dataset region](https://cloud.google.com/bigquery/docs/locations). Also if the previous step showed a numeric value, make sure to un-comment the variable `create_access_context_manager_access_policy = false`. See the shared folder [README.md](./envs/shared/README.md) for additional information on the values in the `terraform.tfvars` file.
```
[the remark this answers is not in the document]
Comment (daniel-cit) [reply 1-org/README.md:43]: Hi @bharathkkb this change would be to replace `terraform_sa_email` with `terraform_service_account` in all the code?

This change would touch these files:

- terraform-example-foundation/0-bootstrap/outputs.tf
- terraform-example-foundation/0-bootstrap/README.md
- terraform-example-foundation/0-bootstrap/modules/jenkins-agent/README.md
- terraform-example-foundation/0-bootstrap/modules/jenkins-agent/variables.tf
- terraform-example-foundation/1-org/README.md
- terraform-example-foundation/test/network_test_prepare.sh
- terraform-example-foundation/test/fixtures/bootstrap/outputs.tf
- terraform-example-foundation/test/fixtures/envs/main.tf
- terraform-example-foundation/test/fixtures/envs/variables.tf
- terraform-example-foundation/test/fixtures/networks/main.tf
- terraform-example-foundation/test/fixtures/networks/variables.tf
- terraform-example-foundation/test/fixtures/org/main.tf
- terraform-example-foundation/test/fixtures/org/variables.tf
- terraform-example-foundation/test/fixtures/projects/main.tf
- terraform-example-foundation/test/fixtures/projects/variables.tf
- terraform-example-foundation/test/fixtures/shared/main.tf
- terraform-example-foundation/test/fixtures/shared/variables.tf
- terraform-example-foundation/test/integration/bootstrap/inspec.yml
- terraform-example-foundation/test/integration/bootstrap/controls/gcp_bootstrap.rb
- terraform-example-foundation/test/setup/outputs.tf

OR 

Would it be to duplicate the terraform output in the `0-bootstrap` step like it was done in the test setup `outputs.tf` file

https://github.com/terraform-google-modules/terraform-example-foundation/blob/0d315491bc3ad7b9578b40042ec97907c4b69cd5/test/setup/outputs.tf#L32

```
output "terraform_sa_email" {
  value = google_service_account.int_test.email
}

output "terraform_service_account" {
  value = google_service_account.int_test.email
}
```

[the remark this answers is not in the document]
Comment (bharathkkb) [reply 1-org/README.md:43]: @daniel-cit so anything in `test/*` is not user facing and we can change. Then the only user facing change is in the output in `0-bootstrap` and aligning it with the rest of the inputs makes sense. WDYT?

[the remark this answers is not in the document]
Comment (daniel-cit) [reply 1-org/README.md:43]: SG

**Comment by daniel-cit** [comment]
@bharathkkb I switched base to main branch and rebased the code

**Pull request merged**